framework/web/View.php

```diff
@@ -170,16 +171,6 @@ class View extends \yii\base\View
      * @var \yii\web\AssetManager the asset manager.
      */
     private $_assetManager;
-    /**
-     * @var bool whether to use jQuery for POS_READY and POS_LOAD script positions.
-     *
-     * When `true` (default), maintains backward compatibility by using jQuery wrappers.
-     * When `false`, uses vanilla JavaScript equivalents.
-     *
-     * @since 2.2.0
-     */
-    public bool $useJquery = true;
```

Comment:
That introduces backwards incompatibility. I'd keep the property here, set it to `null` and configure the view to inherit the setting:

```php
protected function shouldUseJquery(): bool
{
    return $this->useJquery ?? Yii::$app->useJquery;
}
```

Reply:
There is no BC here, this is branch 22, i think this way it will be cleaner, without properties in each component.